.eslintrc.yaml

```diff
@@ -1,10 +1,20 @@
+env:
+  browser: true
+  es2022: true
+  jest: true
+  node: true
 extends:
+  - eslint:recommended
+  - plugin:react/recommended
```

Comment:
These two rule sets could or should have already been included. Adding them surfaced lint errors that are fixed in a later commit in this PR.